docs/en/guide/execute/How-To-Execute.md

```diff
@@ -0,0 +1,36 @@
+# SkyWalking Infra E2E Execute Guide
+
+There are two ways to perform E2E Testing:
+1. Command: Suitable for local debugging and operation.
+1. GitHub Action: Suitable for automated execution in GitHub projects.
+
+## Command
+
+Through commands, you can execute a complete Controller.
+
+```shell
+# e2e.yaml configuration file in current directory
+e2e run
+
+# or 
+
+# Specified the e2e.yaml file path
+e2e run -c /path/to/the/test/e2e.yaml
+```
+
+Also, could run the separate step in the command line, these commands are all done by reading the configuration.
+
+```shell
+e2e setup/trigger/verify/cleanup
```

Comment:
```suggestion
e2e setup
e2e trigger
e2e verify
e2e cleanup
```

Reply:
It may be confused with file path, how about changing to this 👆

Reply:
Thx